when-shit-hits-the-fan.adoc

```diff
@@ -387,7 +387,7 @@ the files blkXXXX.dat and blkindex.dat. The reason for downloading the
 blockchain data this way, as opposed to synchronizing from scratch, was
 to reduce network bandwidth bottlenecks.
 
-There was a big caveat with this: all the data users would download from
+There was a big caveat with this: the data users would download from
 knightmb https://bitcoin.stackexchange.com/a/113682/69518[weren't
```

Comment:
Absolutely not sure, but should "weren't" possibly be "wasn't", not that we changed "all the data" to "the data". You decide.

Reply:
As far as I know, "data" is the plural of the Latin word "datum" and thus requires a plural verb, i.e. "weren't" instead of "wasn't". However, English is not as closely linked to Latin as Italian, therefore I've found "data" + singular verb on many occasions, it is widely accepted.
I'd suggest "weren't" but there is no problem if you prefer the other version